log.go

```diff
@@ -98,6 +98,18 @@ func initLogRotator(logFile string) error {
 	return nil
 }
 
+func UseLogger(logger slog.Logger) {
+	log = logger
+	loader.UseLogger(logger)
+	wallet.UseLogger(logger)
+	udb.UseLogger(logger)
+	ticketbuyer.UseLogger(logger)
+	spv.UseLogger(logger)
+	p2p.UseLogger(logger)
+	connmgr.UseLogger(logger)
+	addrmgr.UseLogger(logger)
```

Comment:
Using the same logger for every package will assign whatever tag you give dcrlibwallet to every package. Can you register a logger for each package from godcr?